lib/rss/rss.rb

```diff
@@ -733,7 +733,7 @@ def install_must_call_validator(prefix, uri)
 
       def install_model(tag, uri, occurs=nil, getter=nil, plural=false)
         getter ||= tag
-        if m = self::MODELS.find {|t, u, o, g, p| t == tag and u == uri}
+        if (m = self::MODELS.find {|t, u, o, g, p| t == tag and u == uri})
```

Comment:
```suggestion
        if m = self::MODELS.find {|t, u, o, g, p| t == tag and u == uri}
```

Not in `Style/StringLiterals`.